client/app/app.js

```diff
@@ -219,10 +219,12 @@ angular
             $locationProvider.hashPrefix("");
         }
     ])
+    .constant("URLS_REDIRECTED_TO_DEDICATED", [new RegExp("/useraccount/infos"), new RegExp("/billing/history")])
```

Comment:
It's not only theses 2 urls, we have more urls in billing & user sections

Reply:
@jleveugle you're right, I'm fixing it